etc/leapp/files/repomap.json

```
@@ -3999,6 +4058,415 @@
                     "rhui": "alibaba"
                 }
             ]
+        },
+        {
+            "pesid": "rhel10-BaseOS",
+            "entries": [
+                {
+                    "major_version": "10",
+                    "repoid": "rhel-10-for-aarch64-baseos-beta-rpms",
+                    "arch": "aarch64",
+                    "channel": "beta",
+                    "repo_type": "rpm"
+                },
+                {
+                    "major_version": "10",
+                    "repoid": "rhel-10-for-aarch64-baseos-rpms",
+                    "arch": "aarch64",
+                    "channel": "ga",
+                    "repo_type": "rpm"
+                },
+                {
+                    "major_version": "10",
+                    "repoid": "rhel-10-for-ppc64le-baseos-beta-rpms",
+                    "arch": "ppc64le",
+                    "channel": "beta",
+                    "repo_type": "rpm"
+                },
+                {
+                    "major_version": "10",
+                    "repoid": "rhel-10-for-ppc64le-baseos-rpms",
+                    "arch": "ppc64le",
+                    "channel": "ga",
+                    "repo_type": "rpm"
+                },
+                {
+                    "major_version": "10",
+                    "repoid": "rhel-10-for-s3100x-baseos-beta-rpms",
+                    "arch": "s3100x",
```

```suggestion
                    "repoid": "rhel-10-for-s390x-baseos-beta-rpms",
                    "arch": "s390x",
```

same typo further in the code

---

Yes that's a substition (s/9/10) error, thanks, good catch :D